db/schema.rb

```diff
@@ -753,7 +755,7 @@
     t.datetime "updated_at",             null: false
   end
 
-  add_index "user_preferences", ["user_id", "name"], name: "index_user_preferences_on_user_id_and_name", unique: true, using: :btree
+  add_index "user_preferences", ["user_id"], name: "index_user_preferences_on_user_id", using: :btree
```

Comment:
This seems like it doesn't belong in this PR.

Reply:
https://github.com/tablexi/nucore-open/pull/1247

Reply:
Also, apparently `expires_mins_before` has already snuck into the schema (in https://github.com/tablexi/nucore-open/pull/1229/files#diff-1acd2e7e27a227829d5d14a91c863bb6R551). Is that going to cause an issue?

Reply:
See my comment on #1247, the original is what the schema _should_ be.

The `expires_mins_before` sneaking in shouldn't cause any problems as long as later PRs didn't revert it.